src/util/render-jsonExt.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { TextInput, NumberInput } from '@openimis/fe-core';
+import { FIELD_TYPES } from '../constants';
+
+export const createAdditionalField = (jsonExt) => {
```

Comment:
While this solution looks good, Json_ext field is quite specific solution in openIMIS that is shared between multiple components and for shared parts it should be handled similarly across the application. For instance: Claim_AI makes contribution to json ext of claim by adding specific claim_ai key and put its data in there (to ensure no key conflicts). At first I thought that is should be the case here as well but in benefit we're in fact not using json_ext same as in other parts of IMIS, we utilise this field because it has right type, but in longer term I think we should replace beneficiary json_ext with custom jsonb field.I don't think it's blocking but we need to bring this discussion to the team.